packages/vats/src/vat-mints.js

```diff
@@ -39,6 +39,7 @@ export function buildRootObject() {
      * @param  {[AssetKind?, DisplayInfo?]} issuerArgs
      */
     makeMintAndIssuer: (issuerNameSingular, ...issuerArgs) => {
+      // makeIssuerKit fails upgrade, safe because demo only?
```

Comment:
I'm not sure. I see this in demo and devnet configs but also https://github.com/Agoric/agoric-sdk/blob/bb30947461eae614ff5e4fac3a9e21b927efe3f6/packages/vats/decentral-core-config.json#L35-L38

@michaelfig ?

Reply:
`decentral-core-config.json` is also non-production.  See the `$comment` at the top.

Reply:
This `notForProductionUse` is pretty cool. I added it to the other zoe contracts with `makerIssuerKit` that are supposed to not end up in production.